widget/entry_password.go

```diff
@@ -67,7 +67,7 @@ func (r *passwordRevealerRenderer) Layout(size fyne.Size) {
 
 func (r *passwordRevealerRenderer) MinSize() fyne.Size {
 	iconSize := r.entry.Theme().Size(theme.SizeNameInlineIcon)
-	return fyne.NewSquareSize(iconSize)
+	return fyne.NewSquareSize(iconSize + r.entry.Theme().Size(theme.SizeNameInnerPadding)*2)
```

Comment:
It looks like you're reserving the inner padding here in the actionItem. Should this be applied in the Entry renderer code instead, so that using a standard widget.Icon as an entry actionItem would result in the same layout and spacing as the password entry's revealer?

Reply:
Good point, thanks

Reply:
Actually after working on this it doesn't seem like a good plan - that would mean that either Entry adds big padding around all action items (including buttons etc) or it has a special case for the revealer or peeks into the renderer of an action item to see if it's a button.

Really it belongs here because the tappable area should not have been just the size of an icon. This is a change and so if you're uneasy about this in a point release we could push back to v2.6.0 with a changelog item mentioning that action items match the expected tappable area by adding inner padding around them if just icons?

Reply:
It does seem like something developers should know about, in order to add the padding around their ActionItem icons to match the rendering of the builtin ones (password entry, etc). So maybe it is better in 2.6